NEWS.rst

```diff
@@ -2,6 +2,24 @@
 Release Notes
 =============
 
+----------------------------
+pkgcore 0.12.19 (2023-01-19)
+----------------------------
+
+- domain: fix parsing of multiple USE_EXAPNDs
+  (Arthur Zamarin, #391, https://bugs.gentoo.org/891409)
+
+- portage_conf: add support for make.profile as its own directory, behaving as
+  specialized user profile (Arthur Zamarin, #390)
+
+General:
```

Comment:
I like this, but it's ok with me if we drop the general and just have the layout match this wrt sorting.